scalafix/output/src/main/scala/incompat/ScalafixIncompat9.scala

```diff
@@ -16,6 +16,6 @@ object ScalafixIncompat9 {
     def cleanup[E](e: E): IO[Nothing] = ???
     def acquire[E]: Acquire[E] = ???
 
-    def foo(io: IO[Nothing]): IO[Nothing] = acquire[Nothing].apply[Any](cleanup[Any]).apply[Nothing](io)
+    def foo(io: IO[Nothing]): IO[Nothing] = acquire[Nothing](cleanup[Any]).apply[Nothing](io)
```

Comment:
Not sure what caused this change, but it doesn't seem like it would break between versions 🤔